### PR TITLE
Removed code that prevents albumentations being used during eval

### PR DESCRIPTION
**Pull request by chdre**
Tested on version 8.0.35 on A5500 16GB GPU, 64GB ram, Windows 10 by
```
yolo classify train data=imagenette160 imgsz=160 model=yolov8l-cls.pt epochs=500 batch=512 workers=4 cache=ram
```
Yields the same best top1 accuracy as torchvision transforms og 0.90. To resolve issue with never using albumentations during test/val when augment is set to False; https://github.com/ultralytics/ultralytics/issues/768

### Summary
:recycle::wrench::sparkles:

Refactor data augmentation pipeline for classification tasks. Simplify `ClassificationDataset` class by removing unnecessary condition for `self.album_transforms`.

> _Simplify `augment`_
> _`classify_albumentations`_
> _Handles all cases_

### Walkthrough
*  Simplify data augmentation pipeline for classification tasks by removing redundant condition ([link](https://github.com/ultralytics/ultralytics/pull/943/files?diff=unified&w=0#diff-9e341b944998965225e7749684518d7461f0a0ebc7e6dd53502ca840fb4e686fL204-R204))

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Enhanced ignore patterns in the project's `.gitignore` file. 

### 📊 Key Changes
- Added `/svv`, `/runs`, and `/wandb` to the `.gitignore` list.

### 🎯 Purpose & Impact
- **Purpose:** Keeps temporary and redundant files like model checkpoints, log files, and experiment logs from cluttering the repository or being inadvertently added to version control.
- **Impact:** Improves developer experience by preventing unneeded files from appearing in Git status and commit changes, reducing the risk of committing unnecessary or sensitive data to the repository.

### CODE REVIEW

**Comment by chdre** [comment]
I have read the CLA Document and I sign the CLA

**Comment by glenn-jocher** [comment]
@chdre thanks for the PR! We have this in place to accelerate transforms for validation, as we found that torchvision profiled substantially faster than albumentation on this (same) set of transforms. 

Can you test the speed of the two implementations and let us know if this is not the case? Thank you!

**Comment by chdre** [comment]
@glenn-jocher I checked the wall time by wrapping the following code in time.time():
https://github.com/ultralytics/ultralytics/blob/20fe708f3197ebaaf98c7fa56db5378b26ee27ac/ultralytics/yolo/data/build.py#L117-L118

To call yolo I decided to use validation by the following call
```
model.val(data='imagenette160', imgsz=160, device=0)
```

The measured times where ~0.03s for code prior to PR and ~0.3s for code in this PR. Removing the imports to the top reduced the wall time from ~0.3 to ~0.03, making the albumentations implementation as fast as torchvision. 

Wrapping model.val(args from above) in time.time() resulted in almost identical wall times of ~22s for both cases.

I also tried simply wrapping model.train(args mentioned in the initial comment) in time.time(). I found no substantial difference in wall time; 173s and 178s vs 181s and 175s for two runs of each, latter being the code from this PR. Reported time of 0.043 and 0.044 hrs from logging.

This is fairly simple, but I hope it's useful anyways. If there is any specific you would like me to test, I would be happy to do so, just please point me in the right direction, as I am not that familiar with advanced profiling :)

**Comment by glenn-jocher** [comment]
@chdre ok got it! Maybe something's changed, since before I noticed a difference with i.e. Colab training of MNIST just by looking at the epoch times. I'll re-run my sanity again.
